tests/functional/test_prometheus_juju_exporter.py

```diff
@@ -9,7 +9,8 @@ def test_snap_startup():
     """Check if the snap is in active state."""
     assert (
         check_call(
-            "systemctl is-active --quiet snap.prometheus-juju-exporter.prometheus-juju-exporter.service".split()
+            "systemctl is-active --quiet"
```

Comment:
A space is missing after `--quiet`, which resulted in func test failure (https://github.com/canonical/prometheus-juju-exporter/actions/runs/4308090809/jobs/7513964416)